operator/deploy/crds/csiscaleoperators.csi.ibm.com.crd.yaml

```diff
@@ -151,6 +151,21 @@ spec:
                     type: string
                 type: object
               type: array
+            snapshotter:
```

Comment:
We're going to need to add more to the CSV for descriptors.

Reply:
Ok, will need your help with this.